Foundation/NSLocale.swift

```diff
@@ -177,18 +177,18 @@ public let NSLocaleScriptCode: String = "kCFLocaleScriptCodeKey"
 public let NSLocaleVariantCode: String = "kCFLocaleVariantCodeKey"
 public let NSLocaleExemplarCharacterSet: String = "kCFLocaleExemplarCharacterSetKey"
 public let NSLocaleCalendar: String = "kCFLocaleCalendarKey"
-public let NSLocaleCollationIdentifier: String = "collation"
+public let NSLocaleCollationIdentifier: String = "kCFLocaleCollationIdentifierKey"
 public let NSLocaleUsesMetricSystem: String = "kCFLocaleUsesMetricSystemKey"
 public let NSLocaleMeasurementSystem: String = "kCFLocaleMeasurementSystemKey"
 public let NSLocaleDecimalSeparator: String = "kCFLocaleDecimalSeparatorKey"
 public let NSLocaleGroupingSeparator: String = "kCFLocaleGroupingSeparatorKey"
 public let NSLocaleCurrencySymbol: String = "kCFLocaleCurrencySymbolKey"
-public let NSLocaleCurrencyCode: String = "currency"
-public let NSLocaleCollatorIdentifier: String = "" // NSUnimplemented // NSString
-public let NSLocaleQuotationBeginDelimiterKey: String = "" // NSUnimplemented // NSString
-public let NSLocaleQuotationEndDelimiterKey: String = "" // NSUnimplemented // NSString
-public let NSLocaleAlternateQuotationBeginDelimiterKey: String = "" // NSUnimplemented // NSString
-public let NSLocaleAlternateQuotationEndDelimiterKey: String = "" // NSUnimplemented // NSString
+public let NSLocaleCurrencyCode: String = "kCFLocaleCurrencyCodeKey"
```

Comment:
Added tests. Should be good now. The only thing: I couldn't get access for `kCFLocaleCurrentLocaleDidChangeNotification` from tests. It's accessible from `Foundation`, though. Do you know how to export it properly?